Platform/ArrowlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1545,6 +1545,10 @@ PlatformUpdateAcpiGnvs (
       if (MtlPchNvs->UM0[Index] == 0x1) {
         MtlPchNvs->UC0[Index] = MtlPchSerialIoUartPciCfgBase (Index);
       }
+      if (MtlPchNvs->UM0[Index] == 0x3) {
+        /*PchConfig->ReservedMmioBase, MTL_PCH_SERIAL_IO_OFFSET, LpssUartFixedPciCfgOffset*/
+        MtlPchNvs->UC0[Index] = 0xBE000000 + 0x00D10000 + 0xD000 + 0x2000 * Index;
```

Comment:
Is it possible to fetch these values at runtime or use #defines instead of hard-coding numbers?